paddle/fluid/operators/split_op.cc

```diff
@@ -136,6 +145,9 @@ This operator splits the input tensor into multiple sub-tensors.
                  "(int, default 0) "
                  "The axis which the input will be split on.")
         .SetDefault(0);
+    AddAttr<bool>("use_mkldnn",
+                  "(bool, default false) Only used in mkldnn kernel")
+        .SetDefault(false);
```

Comment:
There is no need to add following?
```
AddAttr<std::string>(
      "mkldnn_data_type",
      "(string, default \"float32\"). Data type of mkldnn kernel")
      .SetDefault("float32")
      .InEnum({"float32", "int8", "bfloat16"});
```
then cpu_bfloat16_placement_pass.cc will set it "bfloat16" and then in model inference, bf16 kernel will called?
If without this attributes how does the model perf improved. Just curious.

Reply:
I can only comment that reported 20% is  oneDNN execuition without split mkldnn vs with split mkldnn.  Performance of bf16 was not evaluated according to my knwoledge.